src/main/modules/i18next/index.ts

```diff
@@ -0,0 +1,27 @@
+const i18next = require('i18next');
```

Comment:
should we stick with imports?

I think that translates to: `import * as i18next from 'i18next';`

Reply:
Done